app/src/main/AndroidManifest.xml

```diff
@@ -19,63 +19,67 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="org.apache.taverna.mobile">
```

Comment:
Revert all unwanted changes. Change only that line which is required

Reply:
Done! 👍